gapic/api/artman_longrunning.yaml

```diff
@@ -1,5 +1,5 @@
 common:
-  api_name: google-longrunning
+  api_name: google-longrunning-v1
```

Comment:
v1 appears in the REST interface: https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L43